info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -54,17 +59,17 @@
  * article information, and storing them in the database.
  */
 public class InfoCompiler {
-  private final static String ELECTION_QUERY_URL =
+  private static final String ELECTION_QUERY_URL =
       String.format("https://www.googleapis.com/civicinfo/v2/elections?key=%s",
                     Config.CIVIC_INFO_API_KEY);
-  private final static String VOTER_INFO_QUERY_URL =
+  private static final String VOTER_INFO_QUERY_URL =
       String.format("https://www.googleapis.com/civicinfo/v2/voterinfo?key=%s",
                     Config.CIVIC_INFO_API_KEY);
+  private static final Pattern STATE_PATTERN = Pattern.compile(".*state:(..).*");
   private Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
   private List<String> electionQueryIds = new ArrayList<>();
-  // For testing purposes (not to add too much information to the database).
-  // Will include all 50 states.
-  private List<String> states = Arrays.asList("NY");
+  // List of U.S. street addresses that theoretically cover the entire U.S.
+  private List<String> addresses;
```

Comment:
If you're going to initialize electionQueryIds, you should also initialize addresses.

Reply:
`addresses` is getting initialized to a certain size later [here](https://github.com/googleinterns/voter-central/blob/kathy_InfoCompiler_home_address/info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java#L100)
Technically `electionQueryIds` is later initialized as well, but it is first initialized to an empty list, because the second step of gathering information utilizes `electionQueryIds`. If the user doesn't call the first step before calling the second step, there will be an error. However, the user should know to call the first step and then the second step. Should I remove the empty list initialization for `electionQueryIds` to be consistent with `addresses`?

Reply:
Discussed offline

Reply:
Removed initialization for `electionQueryIds`. Hid two steps in compiling information from the user. Write tests for `InfoCompiler.compileInfo`, which executes the entire process of compiling all information: elections, positions and candidates information. Add test for the test election 2000.
Commit: https://github.com/googleinterns/voter-central/pull/26/commits/5543ba9280f685d8a33cf87637ba0fc1987e7844